frontend/src/common/get24HourTime.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @param eventStartTime date time number
+ * @returns fomatted time in form of HH:MM:SS  (eg, 19:00:00 or 12:10:20)
```

Comment:
pls update comment!

Reply:
Resolved in commit 4c7366045b4b24d42221ea1f482ddfc9977ba8c7